crates/ruff_linter/src/rules/ruff/rules/falsy_dict_get_fallback.rs

```diff
@@ -0,0 +1,179 @@
+    }
+}
+
+pub(crate) fn falsy_dict_get_fallback(checker: &mut Checker, expr: &Expr) {
+    // Check if the expression is a call
+    let Expr::Call(ExprCall { func, .. }) = expr else {
+        return;
+    };
+
+    // Check if the function being called is an attribute (e.g. `dict.get`)
+    let Expr::Attribute(ExprAttribute { value, attr, .. }) = &**func else {
+        return;
+    };
+
+    // Ensure the method called is `get`
+    if attr != "get" {
+        return;
+    }
+
+    // Check if the object is a dictionary using the semantic model
+    if let Expr::Name(expr_name) = &**value {
+        if !is_known_to_be_of_type_dict(checker.semantic(), expr_name) {
+            return;
+        }
+    } else {
+        return;
+    }
+
+    let Expr::Call(ExprCall { arguments, .. }) = expr else {
+        return;
+    };
+
+    // Check if a fallback arg is provided
+    if arguments.args.len() < 2 {
+        return;
+    }
+
+    // Get the fallback argument
+    let fallback_arg = &arguments.args[1];
+
+    // Define what is considered a falsy value
+    let is_falsy = is_falsy_fallback(fallback_arg);
+
+    if is_falsy {
+        let diagnostic = Diagnostic::new(FalsyDictGetFallback, fallback_arg.range());
+
+        if let Some(arg) = arguments.args.get(1) {
+            if let Some(prev_arg) = arguments.args.first() {
+                // Start from the end of the first argument (which is the key)
+                let start = prev_arg.range().end();
+                // End at the end of the fallback argument
+                let end = arg.range().end();
+                // Create an edit that deletes from the comma to the end of the fallback
+                let edit = Edit::deletion(start, end);
+
+                // Determine applicability based on the presence of comments
+                let comment_ranges = checker.comment_ranges();
+                let applicability = if comment_ranges.intersects(TextRange::new(start, end)) {
+                    Applicability::Unsafe
+                } else {
+                    Applicability::Safe
+                };
+
+                // Create an automatic fix with the deletion edit and appropriate applicability
+                let fix = ruff_diagnostics::Fix::applicable_edit(edit, applicability);
+
+                // Attach the fix to the diagnostic and add it to the checker's diagnostics
+                checker.diagnostics.push(diagnostic.with_fix(fix));
+                return;
+            }
+        }
+
+        // If unable to determine the arguments correctly, push the diagnostic without a fix
+        checker.diagnostics.push(diagnostic);
+    }
+}
+
+/// Determines if the given expression is a falsy fallback value.
+///
+/// A falsy fallback is one of the following:
+/// 1. `False`
+/// 2. `""`
+/// 3. `[]` or `list()`
+/// 4. `{}` or `dict()`
+/// 5. `set()`
+/// 6. `0`
+/// 7. `0.0`
+/// 8. `None`
+fn is_falsy_fallback(expr: &Expr) -> bool {
```

Comment:
Can we reuse `Truthiness` instead of rolling our own helper here? 

https://github.com/astral-sh/ruff/blob/de62e39eba6553545fa466f709c9a75e88bda9f0/crates/ruff_python_ast/src/helpers.rs#L1127